src/Api/Commands.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace Oveleon\YouTrack\Api;
+
+class Commands extends AbstractApi
+{
+
+    public function command($parameters)
+    {
+        $this->addFields(
+            []
+        );
```

Comment:
I think it would be better if the default fields are implemented via the `defaultFields` method like all others, even if there are none like:
```php
protected function defaultFields(): self
{
    $this->addFields([]);

    return $this;
}